modules/ROOT/pages/hyperforce-configuration.adoc

```diff
@@ -0,0 +1,33 @@
+= Configure Hyperforce in Anypoint Studio
+
+Salesforce Hyperforce (Hyperforce) is a modern, scalable, and secure infrastructure designed to support your MuleSoft application deployments. Hyperforce enables you to deliver Mule apps rapidly and reliably to locations worldwide, giving you more choice and control over data residency. 
```

Comment:
Is this a standard/shared intro?

I think we should avoid "designed to" - can we be more definitive? 
Is there a way to flip it to avoid "enables" (and put the user task first)?

Reply:
Yes! This intro comes from the Hyperforce home page.
Does this wording make it better?

Reply:
Nice rewrite! 

Can we make it a partial and include it in all the product HF pages?